pages/code-review.md

```diff
@@ -114,12 +149,26 @@ When reviewing code, you should be able to check off each of the following:
     - [ ] Do the changes address the project's needs?
     - [ ] Do the changes respect the project's existing style?
     - [ ] Does the new code avoid reproducing existing functionality?
-    - [ ] Are functions as simple as possible?
+    - [ ] Are functions/classes as simple as possible?
     - [ ] Is the code as efficient as possible?
-    - [ ] Is the usage of each function clear?
+    - [ ] Is the usage of each function/class clear?
     - [ ] Have edge cases been considered and tested for?
-
-Use this bookmarklet code to add this checklist to your Pull Request comments: [Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
+    - [ ] Does the code represent a logical unit of work?
+    - [ ] Are there any glaring syntax errors that were missed?
+    - [ ] Are language constructs being utilized properly?
+    - [ ] Are there no unecessary premature optimizations?
+    - [ ] Are any frameworks/libraries being used leveraged properly?
+    - [ ] Are the changes clean and would they pass through a linter?
+    - [ ] Are there useful comments/documentation where needed?
+    - [ ] Is the commit message well-formed with succinct details about
+          what the change is intended for?
```

Comment:
I disagree with @cmc333333. I think detailed commit messages are extremely important. I strongly believe that every commit message must explain what the change is and why the change is necessary. Some changes will require more detail than others. A small typo fix, for example, can probably be described in just the commit title. Other more substantial changes need justification so that anyone who comes across that commit at any point in time can easily understand what happened and why.

I also believe in adhering to style rules for commit messages, such as subjects no longer than 50 characters, using the imperative mood in the subject, no period at the end of the subject.

Here are some relevant links:
http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
http://chris.beams.io/posts/git-commit/
https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message